cypress/integration/chi-vue/data-table.spec.js

```diff
@@ -418,9 +486,29 @@ describe('Data Table', () => {
       });
     });
 
-    it.skip('should trigger the appropriate event', () => {
-      // TODO: add test once workaround is found.
-      const spy = cy.spy();
+    it('should trigger the appropriate event', () => {
```

Comment:
I suggest splitting this into 2 cases: 1 for expansion, 1 for collapsing.
Please use more descriptive titles, for example: `Should trigger the chiRowExpanded event`